src/net35/Radical/Observers/BrokerObserver.cs

```diff
@@ -77,9 +80,9 @@ public MessageBrokerMonitor( IMessageBroker broker )
         /// </summary>
         /// <typeparam name="TMessage">The type of the message.</typeparam>
         /// <returns>This monitor instance.</returns>
-        public MessageBrokerMonitor WaitingFor<TMessage>() where TMessage : class
+        public MessageBrokerMonitor WaitingFor<TMessage>( InvocationModel invocationModel = InvocationModel.Default ) where TMessage : class
```

Comment:
> Unfortunately you need to recompile in order to be able to call that method the methods

really?! OMG, I was thinking that the compiler simply splits in two methods.

but I can add another overload without breaking change isn't it?